app/views/stats/_barchart.html.erb

```diff
@@ -0,0 +1,77 @@
+
+<script src="https://cdn.jsdelivr.net/npm/apexcharts"></script>
+<script>
+var questions = [<%= @questions.present? ? @questions : 0 %>];
+var answers = [<%= @answers %>];
+var unanswered = [ questions[0] - answers[0] ];
```

Comment:
Oh I see you do the subtraction here...my bad

Reply:
Yep there it is! :)